pywbem_mock/_dmtf_cim_schema.py

```diff
@@ -421,6 +443,58 @@ class mof compiles
             :term:`string`: Valid MOF containing pragma statements for
             all of the classes in `schema_classes`.
 
+        Raises:
+            ValueError: If any of the classnames in `schema_classes` are not in
+            the DMTF CIM schema installed
+        """
+        return DMTFCIMSchema.static_build_schema_mof(
+            schema_classes, self.schema_mof_file)
+
+    @staticmethod
+    def static_build_schema_mof(schema_classes, schema_mof_pragma_file):
```

Comment:
How about moving this method to become a global function in the module? It can be used on any CIM schema, not just on the DMTF CIM schema.

Reply:
The advantage is that it would get rid of that stupid name.

DONE